### PR TITLE
Update/fix META file requires

### PR DESCRIPTION
**Pull request by djs55**
This fixes `#require` in the top-level.

### CODE REVIEW

File: pkg/META [hunk 1]
```diff
@@ -6,7 +6,7 @@ package "vfs" (
  directory = "vfs"
  description = "Datakit's VFS description"
  version = "%%VERSION%%"
- requires = ""
+ requires = "fmt astring lwt cstruct logs unix sexplib result"
```
Comment (talex5) [inline pkg/META]: Why does vfs depend on Unix?

Comment (djs55) [inline pkg/META]: Not sure, I got this from

Comment (djs55) [inline pkg/META]: Maybe it's a false positive of some kind? When I comment code out, it disappears after I remove all references to `Cstruct`.

Comment (djs55) [inline pkg/META]: I've removed `unix` and `sexplib` and it seems to still work in `utop`. I clearly don't understand what `ocamlobjinfo` does.

**Pull request merged**